lib/realtime.nit

```diff
@@ -181,4 +182,10 @@ class Clock
 		dt.free
 		return r
 	end
+
+	redef fun finalize_once
```

Comment:
What is the problem? The GC calls right before deallocation so the local attributes are safe.